### DISCUSSION TITLE
How to get user description and custom status?

### DISCUSSION THREAD

**Discussion by q1zZu** [discord-jda/JDA#2357] [Questions and Help] [answered]
Hi, how to get user description ("About me") and custom status?

![kitemtme](https://user-images.githubusercontent.com/80703648/209238013-cc32c8dd-46a3-4a6b-af50-10ca508fd782.png)

**Comment by MinnDevelopment** [comment, marked as answer]
You can get the custom status through [Member#getActivities](https://docs.jda.wiki/net/dv8tion/jda/api/entities/Member.html#getActivities()). The about me is inaccessible to bots.
